html/changelogs/dansemacabre-vending.yml

```diff
@@ -38,4 +38,4 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - backend: "Fixed some backend voting problems."
+  - imageadd: "Replaces the sprites for the soda vending machine, stools, and grilles."
```

Comment:
Interesting changelog choice. You should delete this, make a copy from the example.yml and edit that.

Reply:
As I said, this was github being weird, not necessarily my fault. I'll see if I can fix it, though.